tst/EnergyPlus/unit/PackagedTerminalHeatPump.unit.cc

```diff
@@ -117,498 +117,503 @@ using namespace EnergyPlus::ZoneTempPredictorCorrector;
-    EXPECT_EQ(sizingAirflowCapacityRatio, state->dataVariableSpeedCoils->VarSpeedCoil(2).MSRatedAirVolFlowPerRatedTotCap(9));
-
-    state->dataFans->fans(1)->set_size(*state);
-    // the fan vol flow rate should equal the max of cooling and heating coil flow rates
-    Real64 maxCoilAirFlow =
-        max(state->dataVariableSpeedCoils->VarSpeedCoil(1).RatedAirVolFlowRate, state->dataVariableSpeedCoils->VarSpeedCoil(2).RatedAirVolFlowRate);
-    EXPECT_NEAR(state->dataFans->fans(1)->maxAirFlowRate, maxCoilAirFlow, 0.000001);
-    EXPECT_NEAR(state->dataFans->fans(1)->maxAirFlowRate, max(thisSys.m_MaxCoolAirVolFlow, thisSys.m_MaxHeatAirVolFlow), 0.000001);
-
-    // Also set BeginEnvrnFlag so code is tested for coil initialization and does not crash
-    state->dataGlobal->BeginEnvrnFlag = true;
-    thisSys.initUnitarySystems(*state, 0, firstHVACIteration, 0.0);
-
-    // check that an intermediate speed has the correct flow ratio
-    Real64 refAirflowRatio = 0.530468926 / 0.891980668; // speed 4 reference cooling data and full flow rate at speed 9
-    Real64 expectedAirFlowRate = refAirflowRatio * thisSys.m_MaxCoolAirVolFlow;
-    EXPECT_NEAR(expectedAirFlowRate, thisSys.m_CoolVolumeFlowRate[4], 0.0000001);
-    EXPECT_NEAR(expectedAirFlowRate, 0.5947088, 0.000001); // DIFF from PTHP
-
-    refAirflowRatio = 0.530468926 / 0.891980668; // speed 4 reference heating data and full flow rate at speed 9
-    expectedAirFlowRate = refAirflowRatio * thisSys.m_MaxHeatAirVolFlow;
-    EXPECT_NEAR(expectedAirFlowRate, thisSys.m_HeatVolumeFlowRate[4], 0.2); // DIFF from PTHP
-    EXPECT_NEAR(expectedAirFlowRate, 0.5947, 0.0001);                       // DIFF from PTHP
-
-    // DIFF - comments for PTHP Variable Speed coil sizing
-    // #6028 child components not sizing correctly on air flow rate
-    // VS coils set SystemAirFlow to true and AirVolFlow to a value.
-    // all PTUnits set CoolingAirFlow and HeatingAirFlow, and CoolingAirVolFlow and HeatingAirVolFlow
-    // UnitarySystem now tracks the way PT units sized coils, which method is correct is for another day
-    EXPECT_FALSE(state->dataSize->ZoneEqSizing(1).SystemAirFlow);
-    EXPECT_NEAR(state->dataSize->ZoneEqSizing(1).AirVolFlow, state->dataVariableSpeedCoils->VarSpeedCoil(1).RatedAirVolFlowRate, 0.000001);
-    EXPECT_FALSE(state->dataSize->ZoneEqSizing(1).CoolingAirFlow);
-    EXPECT_FALSE(state->dataSize->ZoneEqSizing(1).HeatingAirFlow);
-    EXPECT_EQ(state->dataSize->ZoneEqSizing(1).CoolingAirVolFlow, thisSys.m_MaxCoolAirVolFlow);
-    EXPECT_LT(state->dataSize->ZoneEqSizing(1).HeatingAirVolFlow, thisSys.m_MaxHeatAirVolFlow);
-    EXPECT_EQ(state->dataFans->fans(1)->maxAirFlowRate, state->dataSize->ZoneEqSizing(1).AirVolFlow);
-    EXPECT_EQ(state->dataFans->fans(1)->maxAirFlowRate,
-              max(state->dataSize->ZoneEqSizing(1).CoolingAirVolFlow, state->dataSize->ZoneEqSizing(1).HeatingAirVolFlow));
-}
+// TEST_F(EnergyPlusFixture, PackagedTerminalHP_VSCoils_Sizing)
```

Comment:
There was a _**famous**_ test here that was disabled, and hadn't been run in a long time.  (PackagedTerminalHP_VSCoils_Sizing)  Since ctest is more aggressively reporting out disabled tests, I commented it out.  Happy to get it back in if someone wants to spend time investigating the failure.